app/src/main/java/com/codesoom/assignment/product/application/ProductService.java

```diff
@@ -0,0 +1,17 @@
+package com.codesoom.assignment.product.application;
+
+import com.codesoom.assignment.product.ui.dto.ProductResponseDto;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class ProductService {
+    public List<ProductResponseDto> getProducts() {
+        return null;
+    }
```

Comment:
이렇게 주석을 적어 주시면 리뷰하는 사람이 읽기 편할 거에요.
```suggestion
    public List<ProductResponseDto> getProducts() {
        // TODO: 응답용 상품 리스트를 리턴하세요
        return null;
    }
```

Reply:
TDD로 작성하느랴 메서드만 놔두었는데, 좋은 습관인 것같네요!